yarn.lock

```diff
@@ -6724,7 +6724,7 @@ js-yaml@^3.13.1:
     argparse "^1.0.7"
     esprima "^4.0.0"
 
-jsc-android@^250230.2.1:
```

Comment:
I see you removed an entry from `package.json` it is also better to revert this file as well @hysekbutt, since nix relies on `yarn.lock` to generate dependencies :)

Reply:
Removed from `yarn.lock`

Reply:
I already removed the changes of `yarn.lock` file